shell/edit/provisioning.cattle.io.cluster/tabs/Basics.vue

```diff
@@ -119,12 +119,20 @@ export default {
     }
   },
 
+  data() {
+    return { showEnablingComplianceWarning: false };
+  },
+
   watch: {
     selectedVersion(neu, old) {
       if (neu?.value !== old?.value && this.ciliumIpv6) {
         // Re-assign so that the setter updates the structure for the new k8s version if needed
         this.ciliumIpv6 = !!this.ciliumIpv6;
       }
+    },
+
+    'agentConfig.profile'(newValue) {
```

Comment:
if the user goes back to the original value (no change needed) we still show the banner